pkg/node/node.go

```diff
@@ -760,14 +760,16 @@ func NewBee(addr string, publicKey *ecdsa.PublicKey, signer crypto.Signer, netwo
 	var apiService api.Service
 	if o.APIAddr != "" {
 		// API server
+		var chunkC <-chan *pusher.Op
 		feedFactory := factory.New(ns)
 		steward := steward.New(storer, traversalService, retrieve, pushSyncProtocol)
-		apiService = api.New(tagService, ns, multiResolver, pssService, traversalService, pinningService, feedFactory, post, postageContractService, steward, signer, authenticator, logger, tracer, api.Options{
+		apiService, chunkC = api.New(tagService, ns, multiResolver, pssService, traversalService, pinningService, feedFactory, post, postageContractService, steward, signer, authenticator, logger, tracer, api.Options{
 			CORSAllowedOrigins: o.CORSAllowedOrigins,
 			GatewayMode:        o.GatewayMode,
 			WsPingPeriod:       60 * time.Second,
 			Restricted:         o.Restricted,
 		})
+		pusherService.SetApiC(chunkC)
```

Comment:
So context from my previous context regarding enqueuing a separate channel for every putter...

I find this a bit confusing. We call this SetApiC() function here. This looks like it will tell the pusher service to start taking chunks from this channel. But what we are doing is creating a separate goroutine to read from this channel.

So the SetApiC function can be called multiple times, and it is not being used that way. There is nothing wrong here from a code perspective, just less clarity on the use of this.

Reply:
I think that the `SetApiC` can be renamed into something more like `FeedFrom(chan *Op)` and this would resolve the contention here. Am I correct?

Reply:
`AddFeed` sounds better. As the interface to pusher specifies that more feeds can possibly be added.